cpp/src/gandiva/function_registry_string.cc

```diff
@@ -163,6 +163,12 @@ std::vector<NativeFunction> GetStringFunctionRegistry() {
       NativeFunction("rpad", {}, DataTypeVector{utf8(), int32()}, utf8(),
                      kResultNullIfNull, "rpad_utf8_int32", NativeFunction::kNeedsContext),
 
+      NativeFunction("regexp_extract", {}, DataTypeVector{utf8(), utf8(), int32()},
+                     utf8(), kResultNullIfNull, "gdv_fn_regexp_extract_utf8_int32",
```

Comment:
Rename the method to "gdv_fn_regexp_extract_utf8_utf8_int32"

Reply:
The changes were applied